### ISSUE TITLE
how to solve the problem of random paras?

### ISSUE THREAD

**Issue by lieliefengzheng** [typicode/json-server#189]
I wanna to mock the testing environment,and faced a problem.When I request a url or api ,the url contains some parameters,like this:localhost:3000/page.api?firstpara=xxx&secondpara=xxx&thirdpara=xxx. xxx is a random data, I wanna to ignore the parameters.
I mean the response of  this url:localhost:3000/page.api?firstpara=xxx&secondpara=xxx&thirdpara=xxx  equals the response of this::localhost:3000/page.api.

Is there any solution to this? thx.

**Comment by blairanderson** [comment]
yes in your test using a mock you can start an express server and return anything you want. your test mocks should not rely/depend on `json-server`

**Comment by typicode** [comment]
Hi @lieliefengzheng,

If params don't match a "path" in your data, they should be automatically ignored.
You have an example in the changelog file.

In other words, if you don't have firstpara property in your db.json, you shouldn't have problems.

PS: sorry for the lack of examples, I'm replying from my phone.

**Comment by typicode** [comment]
Let me know if that helps.

**Comment by lieliefengzheng** [comment]
yep,it works ! thx.
